tex/artikel/Erstiwochenende.tex

```diff
@@ -6,7 +6,7 @@ \section{Erstiwochenende -- Was dann geschah ist unglaublich}
 
 	% XXX Jedes Jahr Ersti-WE-Termin aktualisieren!
 \textbf{\emph{Hinweis:}
-Das Erstiwochenende findet dieses Semester vom 04.11. bis zum 06.11.\ statt. (Verbindliche) Anmeldung in der Fachschaft. Wir freuen uns auf euch!
+Das Erstiwochenende findet dieses Semester vom 03.11. bis zum 05.11.\ statt. (Verbindliche) Anmeldung in der Fachschaft oder der Webseite. Wir freuen uns auf euch!
```

Comment:
Kann man sich wirklich über eine Website auf für die Ersti-Fahrt anmelden?

Reply:
Stand so in Alexanders Text